meeting-notes/2023/2023-04-11_europe_africa_cc.md

```diff
@@ -0,0 +1,35 @@
+# scikit-image Community Call
+
+- **Time:** Tuesday, Apr 11, 2023, 17:00 – 18:00 UTC
+- **[Join video call via Zoom](https://us06web.zoom.us/j/88060567580?pwd=THRpaWFnSFNwK0Fycy9FVk5RYnV5UT09)**
+- **Notes:** https://hackmd.io/@scientific-python/HJWUBDVNi
+- **[scikit-image Community Calendar](https://scientific-python.org/calendars/skimage.ics)**
+- **[Archive — Meeting Notes](https://github.com/scikit-image/meeting-notes)**
+- **[Code of Conduct](https://scikit-image.org/docs/stable/conduct/code_of_conduct.html)**
+
+**Present:** Jarrod, Stéfan, Lars
+
+**Moderator:** -
+
+
+## Agenda
+
+- scikit-image v0.21
+  - [0.21 milestone](https://github.com/scikit-image/scikit-image/milestone/9): ~42 open
+  - Schedule: rc0 on April 11, final on April 14
```

Comment:
As a matter of policy, can we try to avoid making final releases on Fridays? It's very hard to ensure a release doesn't break *someone*'s code or CI, and it's annoying when this happens right before the weekend.

Reply:
CC @stefanv @jarrodmillman